### PR TITLE
 Improve cmdlet descriptions and parameter descriptions for VM/VMSS creation.

### PR DESCRIPTION
**Pull request by grizzlytheodore**
## Description
 Improve cmdlet descriptions and parameter descriptions for VM/VMSS creation.

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Compute/Compute/help/New-AzVM.md [hunk 1]
```diff
@@ -390,7 +397,16 @@ Accept wildcard characters: False
 
 ### -Credential
 The administrator credentials for the VM.
-
+#### Username
```
Comment (wyunchi-ms) [inline src/Compute/Compute/help/New-AzVM.md:400]: Microsoft docs doesn't support Lv4 header now. Please use another format.

File: src/Compute/Compute/help/New-AzVM.md [hunk 2]
```diff
@@ -57,11 +57,18 @@ New-AzVM [[-ResourceGroupName] <String>] [[-Location] <String>] [-EdgeZone <Stri
...
+- **[Add-AzVMNetworkInterface](https://learn.microsoft.com/en-us/powershell/module/az.compute/add-azvmnetworkinterface)** to set the network profile.<br>
+- **[Set-AzVMOperatingSystem](https://learn.microsoft.com/en-us/powershell/module/az.compute/set-azvmoperatingsystem)** to set the OS profile. <br>
+- **[Set-AzVMSourceImage](https://learn.microsoft.com/en-us/powershell/module/az.compute/set-azvmsourceimage)** to set the source image.<br>
+- **[Set-AzVMOSDisk](https://learn.microsoft.com/en-us/powershell/module/az.compute/set-azvmosdisk)** to set the OS disk(storage profile).<br>
+- **[Get-AzComputeResourceSku](https://docs.microsoft.com/en-us/powershell/module/az.compute/get-azcomputeresourcesku)** can also be used to find out available virtual machine sizes for your subscription and region.<br>
+<br>
+The `SimpleParameterSet` provides a convenient method to create a VM by making common VM creation arguments optional.<br>
+<br>
+See [Quickstart: Create a Windows virtual machine in Azure with PowerShell](https://learn.microsoft.com/en-us/azure/virtual-machines/windows/quick-create-powershell) for tutorial. <br>
```
Comment (grizzlytheodore) [inline src/Compute/Compute/help/New-AzVM.md:71]: For some reason, this hyper link is not working. 
The ones above are working however. 
Any idea why? @wyunchi-ms 
see here https://github.com/Azure/azure-powershell/blob/bb0355f075659979c3be620af0a6342a2112d404/src/Compute/Compute/help/New-AzVM.md#description

File: src/Compute/Compute/help/New-AzVM.md (hunk 1 above)
Comment (grizzlytheodore) [reply src/Compute/Compute/help/New-AzVM.md:400]: how about lv 5 and on?

Comment (grizzlytheodore) [reply src/Compute/Compute/help/New-AzVM.md:400]: @wyunchi-ms

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**